### PR TITLE
Watch implementation

### PR DESCRIPTION
**Pull request by sudhirt4**
Watch implementation

### CODE REVIEW

File: app/controllers/api/v1/books_controller.rb [hunk 1]
```diff
@@ -50,7 +50,7 @@ def authors
   end
 
   def recent_books
-    @books = Book.last(5)
+    @books = Book.last(7)
```
Comment (lf-kailashraj) [inline app/controllers/api/v1/books_controller.rb]: Y U No use constants?

**Comment by gurungsuman15** [comment]
:+1:

File: public/application/books/books.controller.js [hunk 2]
```diff
@@ -32,13 +34,18 @@
     });
 
     $scope.searchBookCategory = '';
+    $scope.searchBookName = '';
     $scope.updateFilter = function () {
```
Comment (niraj-rajbhandari) [inline public/application/books/books.controller.js]: Is this function called in view ? if not may be we can create a local (private) function.

File: public/application/books/books.controller.js [hunk 3]
```diff
@@ -32,13 +34,18 @@
...
-      var filtered = $filter('bookFilter')($scope.books, $scope.searchBookCategory);
-      $scope.filteredBooks = filtered;
+      var filtered = $filter('bookFilter')($scope.allBooks, $scope.searchBookCategory);
+      filtered = $filter('filter')(filtered, {name: $scope.searchBookName});
+      $scope.books = filtered;
```
Comment (niraj-rajbhandari) [inline public/application/books/books.controller.js]: May be we can directly assigned the filtered list to the scope?

File: public/application/books/books.controller.js [hunk 4]
```diff
@@ -132,16 +158,20 @@
     BooksFactory.getBook($stateParams.bookId)
     .success(function (data) {
       $scope.book = data;
+      console.log($scope.book)
```
Comment (niraj-rajbhandari) [inline public/application/books/books.controller.js]: Please remove unwanted code. ;)

**Comment by niraj-rajbhandari** [comment]
:+1:

**Pull request merged**